public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -12,7 +12,16 @@ interface LogsPanelProps extends PanelProps<Options> {}
 export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
```

Comment:
In LogsPanel, this is not going to work because you haven't add the switch to control this setting. The switch should be added in `public/app/plugins/panel/logs/module.tsx`
![image](https://user-images.githubusercontent.com/30407135/125456044-e60736f8-3e06-48f3-b39c-ec106a4d19f4.png)

Reply:
Oh i see, i had no idea! Thanks for the information 😄